src/Grid/Grid.js

```diff
@@ -9,18 +9,20 @@ import {
 import Box from '../Box';
 import createWithComponent from '../utils/createWithComponent';
 
-const Grid = styled(createWithComponent(Box, 'div'))`
+const Grid = styled(
+  createWithComponent(Box, 'div', {
+    propTypes: {
+      ...gridGap.propTypes, // eslint-disable-line react/forbid-foreign-prop-types
```

Comment:
I could see that this eslint rule will need to be disabled in quite a few places. Shall we just disable it globally?

Reply:
Good call. I think that makes sense. I'll add it to the .eslintrc